[2025-05-25T05:24:08.002+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-25T05:24:09.632+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2025-01-21T00:00:00+00:00 [queued]>
[2025-05-25T05:24:10.118+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2025-01-21T00:00:00+00:00 [queued]>
[2025-05-25T05:24:10.120+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-05-25T05:24:11.789+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): generate_file> on 2025-01-21 00:00:00+00:00
[2025-05-25T05:24:12.114+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=615) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-25T05:24:12.182+0000] {standard_task_runner.py:72} INFO - Started process 660 to run task
[2025-05-25T05:24:12.193+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_file_to_psql', 'generate_file', 'scheduled__2025-01-21T00:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/data_quality.py', '--cfg-path', '/tmp/tmpfg01rtsz']
[2025-05-25T05:24:12.381+0000] {standard_task_runner.py:105} INFO - Job 70: Subtask generate_file
[2025-05-25T05:24:12.915+0000] {task_command.py:467} INFO - Running <TaskInstance: load_file_to_psql.generate_file scheduled__2025-01-21T00:00:00+00:00 [running]> on host b4cf172ed0b9
[2025-05-25T05:24:26.625+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='https://t.me/Shust_DE' AIRFLOW_CTX_DAG_OWNER='@Shust_DE' AIRFLOW_CTX_DAG_ID='load_file_to_psql' AIRFLOW_CTX_TASK_ID='generate_file' AIRFLOW_CTX_EXECUTION_DATE='2025-01-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-21T00:00:00+00:00'
[2025-05-25T05:24:26.673+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-25T05:24:28.326+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-25T05:24:28.674+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-25T05:24:28.689+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_file_to_psql, task_id=generate_file, run_id=scheduled__2025-01-21T00:00:00+00:00, execution_date=20250121T000000, start_date=20250525T052409, end_date=20250525T052428
[2025-05-25T05:24:29.599+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-25T05:24:31.132+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-05-25T05:24:45.497+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-25T05:24:45.519+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
